docs/source/models/gbd2017/concept_models/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _2017_concept_models:
+
+=========================
+Simulation Concept Models
+=========================
+
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   */*
```

Comment:
Use */index here.  You'll pick up a bunch of files you don't want with the raw glob.

Reply:
I incorporated this change in the "develop" branch where I'm writing the documentation: https://github.com/ihmeuw/vivarium_research/tree/develop/concept_model/lsff